packages/angular/cli/commands/generate.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/generate.json",
-  "description": "Generates and/or modifies files based on a schematic.",
-  "$longDescription": "",
+  "description": "Creates, initializes, and modifies files to create a generic version of an app or app element in the given or default project, according to the given schematic.",
+  "$longDescription": "Takes the name of the project, as specified in the \"projects\" section of the angular.json workspace configuration file. \n When a project name is not supplied, the configured default project of the workspace is used.",
```

Comment:
The generate command actually doesn't take the name of a project in the same way as the other commands.  It is currently an outlier and we've been discussing the best path forward to potentially remedy the situation.

Reply:
`@longDescription` also needs to point to a separate file.

Reply:
No long description now. 
I'm also trying to formulate better boilerplate for the ones that do take an optional project. Maybe we can add it to definitions.json